libraries/chain/include/eosio/chain/block.hpp

```diff
@@ -129,3 +144,20 @@ FC_REFLECT_DERIVED(eosio::chain::transaction_receipt, (eosio::chain::transaction
 FC_REFLECT(eosio::chain::additional_block_signatures_extension, (signatures));
 FC_REFLECT(eosio::chain::quorum_certificate_extension, (qc));
 FC_REFLECT_DERIVED(eosio::chain::signed_block, (eosio::chain::signed_block_header), (transactions)(block_extensions) )
+
+namespace fc::raw {
+   template <typename Stream>
+   void unpack(Stream& s, eosio::chain::signed_block& v) {
+      try {
+         if constexpr (requires { s.extract_mirror(); }) {
+            fc::reflector<eosio::chain::signed_block>::visit( fc::raw::detail::unpack_object_visitor<Stream, eosio::chain::signed_block>( v, s ) );
+            v.packed_block = s.extract_mirror();
+         } else {
+            fc::datastream_mirror<Stream> ds(s, sizeof(eosio::chain::signed_block) + 4096);
```

Comment:
I don't think the `sizeof(eosio::chain::signed_block)` is very significant since most of the memory used is likely in the `deque<transaction_receipt>   transactions; ` whose `sizeof()` is probably around 16. I think I would just simplify to just 4096, but it probably doesn't matter much and is fine as is.